testGameSearch/GameService.swift

```diff
@@ -19,14 +19,16 @@ public class GameService {
     }
     */
     
-    func getGames(game: String, completion: @escaping (Game?) -> ()) {
+    func getGames(gameName: String, completion: @escaping (Game?) -> ()) {
```

Comment:
I tried to rename strings to gameName for maximum clarity as to which game is a string and which game is a struct

Reply:
that makes sense. it was getting convoluted